dockstore-webservice/src/main/java/io/dockstore/webservice/resources/DockerRepoResource.java

```diff
@@ -166,42 +170,38 @@ public Container getContainer(@ApiParam(hidden = true) @Auth Token authToken,
 
         return c;
     }
-    
+
     @PUT
     @Timed
     @UnitOfWork
     @Path("/{containerId}/labels")
-    @ApiOperation(value = "Update the labels linked to a container.",
```

Comment:
@oicr-vchung Ok, use the code formatter, I've added a link to the instructions to the readme, that file should also be compatible with Eclipse, Intellij, and NetBeans (with formatter plugin)